src/Modules/Server.hpp

```diff
@@ -28,6 +28,7 @@ class Server : public Module {
 
     CGlobalVars* gpGlobals = nullptr;
     CEntInfo* m_EntPtrArray = nullptr;
+    CEntInfo* portalGun = nullptr;
```

Comment:
Note that if we use LookupEntity you have to change the type to `IHandleEntity*`.